### PR TITLE
Added subjects table to database

### PR DESCRIPTION
**Pull request by shravnages**

### CODE REVIEW

File: app/Lesson.php [hunk 1]
```diff
@@ -14,4 +14,8 @@ public function tutor()
   {
     return $this->belongsTo('App\Tutor', 'tutor_id', 'user_id');
   }
+
+  public function subject() {
+    return $this->belongsTo('App\Subjects', 'subject_id', 'id');
```
Comment (jasonlipo) [inline app/Lesson.php]: `return $this->belongsTo('App\Subject');`
Check by using:
`php artisan tinker`
`Lesson::find(1)->subject`

File: database/seeds/LessonsSeeder.php [hunk 2]
```diff
@@ -1,30 +1,32 @@
 <?php
 
 use Illuminate\Database\Seeder;
+use App\Subjects;
```
Comment (jasonlipo) [inline database/seeds/LessonsSeeder.php]: `use App\Subject;`

File: app/Subject.php [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Subject extends Model
+{
+  public function lessons() {
+    return $this->hasMany('App\Lesson', 'subject_id', 'id');
```
Comment (jasonlipo) [inline app/Subject.php]: Again just check with tinker

File: database/migrations/2018_06_06_143406_add_subjects_to_lessons_table.php [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+      Schema::dropIfExists('lessons');
```
Comment (jasonlipo) [inline database/migrations/2018_06_06_143406_add_subjects_to_lessons_table.php]: Correct `down()` method

Comment (jasonlipo) [reply database/migrations/2018_06_06_143406_add_subjects_to_lessons_table.php]:
```
Schema::table('lessons', function (Blueprint $table) {
  $table->string('subject');
});
Schema::table('lessons', function (Blueprint $table) {
  $table->dropColumn('subject_id');
  $table->dropForeign('subject_id');
});
```

File: app/Subject.php (hunk 3 above)
Comment (shravnages) [reply app/Subject.php]: Checked! It works.

**Pull request merged**